### ISSUE TITLE
Win32, Win64 binaries in npm

### ISSUE THREAD

**Issue by tracker1** [w1nk/node-odbc#31]
I know the windows support is immature at this point, and not entirely flushed out.. that said, it will be nice to have a target for the npm install for windows that will simply download prebuilt win32/64 binaries instead of requiring a built environment which isn't typical for windows servers.

Nice to know the windows support has continued to develop.

**Comment by wankdanker** [comment]
I agree that this would be nice. I did look in to it out of curiosity
a month back or so and at that time there was no "blessed" way to do it. I
have seen some modules that do it, but there always seemed to be
controversy surrounding the implementation. Or just that binaries shouldn't
be included in npm at all. But, maybe I'm misinformed.

If there is a specification for inclusion of binaries, then yeah, we should
get working on that asap. Otherwise, I personally will wait until there is
a clear spec, someone proposes something concrete or the company I work for
has a need for it.

So, that's where I stand on putting effort into it. I definitely welcome
patches though. :)

Thanks,

Dan

**Comment by tracker1** [comment]
Added an issue for node mainline... <a href="https://github.com/joyent/node/issues/4398">https://github.com/joyent/node/issues/4398</a>

**Comment by bustta** [comment]
Node-pre-gyp seems works. ( https://github.com/mapbox/node-pre-gyp/ )
Maybe it's the twilight for Win binaries of node-odbc?

**Comment by kylecordes** [comment]
This issue is not truly Windows-specific - it would be highly, highly convenient also to ship working binaries on Linux.

**Comment by rhtpandeyIN** [comment]
@wankdanker , Could you please tell me which one is the installer file here, I am sorry I am not clear about it. Just wanted to help to create patch for windows installer binaries.
